initial_object_poses:
  precision_tool:
    target_link: precision_tool_holder_link
  screw_tool_m3:
    target_link: screw_tool_m3_holder_link
  screw_tool_m4:
    target_link: screw_tool_m4_holder_link
  suction_tool:
    target_link: suction_tool_holder_link
#  base:
#    target_link: workbench_holder_link
#  panel_motor:
#    target_link: base/panel_motor_screw_hole_1
#    source_link: panel_motor/bottom_screw_hole_aligner_1
#  panel_bearing:
#    target_link: base/panel_bearing_screw_hole_1
#    source_link: panel_bearing/bottom_screw_hole_aligner_1

placing_parameters:
  precision_tool_holder_link:
    approach_offset:  [-0.002, 0.000, 0.050, 0.0, 0.0, 0.0]
    place_offset:     [ 0.000, 0.000, 0.000, 0.0, 0.0, 0.0]
    departure_offset: [-0.002, 0.000, 0.050, 0.0, 0.0, 0.0]
  screw_tool_m3_holder_link:
    approach_offset:  [ 0.140, 0.000, 0.002, 0.0, 0.0, 0.0]
    place_offset:     [ 0.000, 0.000, 0.000, 0.0, 0.0, 0.0]
    departure_offset: [ 0.140, 0.000, 0.002, 0.0, 0.0, 0.0]
  screw_tool_m4_holder_link:
    approach_offset:  [ 0.140, 0.000, 0.002, 0.0, 0.0, 0.0]
    place_offset:     [ 0.000, 0.000, 0.000, 0.0, 0.0, 0.0]
    departure_offset: [ 0.140, 0.000, 0.002, 0.0, 0.0, 0.0]
  suction_tool_holder_link:
    approach_offset:  [-0.002, 0.000, 0.055, 0.0, 0.0, 0.0]
    place_offset:     [ 0.000, 0.000, 0.000, 0.0, 0.0, 0.0]
    departure_offset: [-0.002, 0.000, 0.055, 0.0, 0.0, 0.0]
  default:
    approach_offset:  [ 0.000, 0.000, 0.050, 0.0, 0.0, 0.0]
    place_offset:     [ 0.000, 0.000, 0.000, 0.0, 0.0, 0.0]
    departure_offset: [ 0.000, 0.000, 0.050, 0.0, 0.0, 0.0]

picking_parameters:
  precision_tool:
    approach_offset:  [-0.007, 0.000,  0.050, 0.0, 15.0, 0.0]
    pick_offset:      [-0.007, 0.000, -0.035, 0.0, 15.0, 0.0]
    departure_offset: [-0.007, 0.000,  0.050, 0.0, 15.0, 0.0]
    speed_fast:       1.0
    speed_slow:       0.1
    gripper_parameters:
      grasp_position:   0.000
      release_position: 0.065
      max_effort:       5.0
  screw_tool_m3:
    approach_offset:  [ 0.140, 0.000, 0.010, 0.0, 15.0, 0.0]
    pick_offset:      [-0.005, 0.000, 0.008, 0.0, 15.0, 0.0]
    departure_offset: [ 0.140, 0.000, 0.010, 0.0, 15.0, 0.0]
    speed_fast:       1.0
    speed_slow:       0.1
    gripper_parameters:
      grasp_position:   0.000
      release_position: 0.035
      max_effort:       5.0
  screw_tool_m4:
    approach_offset:  [ 0.140, 0.000, 0.010, 0.0, 10.0, 0.0]
    pick_offset:      [-0.005, 0.000, 0.008, 0.0, 10.0, 0.0]
    departure_offset: [ 0.140, 0.000, 0.010, 0.0, 10.0, 0.0]
    speed_fast:       1.0
    speed_slow:       0.1
    gripper_parameters:
      grasp_position:   0.000
      release_position: 0.035
      max_effort:       5.0
  suction_tool:
    approach_offset:  [-0.002, 0.000, 0.055, 0.0, 0.0, 0.0]
    pick_offset:      [ 0.000, 0.000, 0.000, 0.0, 0.0, 0.0]
    departure_offset: [-0.002, 0.000, 0.055, 0.0, 0.0, 0.0]
    speed_fast:       1.0
    speed_slow:       0.1
    gripper_parameters:
      grasp_position:   0.000
      release_position: 0.085
      max_effort:       5.0
  base:
    approach_offset:  [0.000, 0.000, 0.060, 0.0, 0.0, 0.0]
    pick_offset:      [0.000, 0.000, 0.002, 0.0, 0.0, 0.0]
    departure_offset: [0.000, 0.000, 0.060, 0.0, 0.0, 0.0]
    speed_fast:       0.1
    speed_slow:       0.1
    gripper_parameters:
      grasp_position:   0.000
      release_position: 0.0402
      max_effort:       5.0
  panel_motor:
    approach_offset:  [0.000, 0.000,  0.060, 0.0, 0.0, 0.0]
    pick_offset:      [0.000, 0.000, -0.020, 0.0, 0.0, 0.0]
    departure_offset: [0.000, 0.000,  0.060, 0.0, 0.0, 0.0]
    speed_fast:       0.1
    speed_slow:       0.1
    gripper_parameters:
      grasp_position:   0.000
      release_position: 0.025
      max_effort:       5.0
  panel_bearing:
    approach_offset:  [0.000, 0.000,  0.060, 0.0, 0.0, 0.0]
    pick_offset:      [0.000, 0.000, -0.020, 0.0, 0.0, 0.0]
    departure_offset: [0.000, 0.000,  0.060, 0.0, 0.0, 0.0]
    speed_fast:       0.1
    speed_slow:       0.1
    gripper_parameters:
      grasp_position:   0.000
      release_position: 0.025
      max_effort:       5.0
  screw_m3:
    approach_offset:  [0.000, 0.000,  0.080, 0.0, 0.0, 0.0]
    pick_offset:      [0.000, 0.000,  0.000, 0.0, 0.0, 0.0]
    departure_offset: [0.000, 0.000,  0.080, 0.0, 0.0, 0.0]
    speed_fast:       1.0
    speed_slow:       0.1
  screw_m4:
    approach_offset:  [0.000, 0.000,  0.080, 0.0, 0.0, 0.0]
    pick_offset:      [0.000, 0.000,  0.000, 0.0, 0.0, 0.0]
    departure_offset: [0.000, 0.000,  0.080, 0.0, 0.0, 0.0]
    speed_fast:       1.0
    speed_slow:       0.1
